packages/firestore/exp/src/api/database.ts

```diff
@@ -332,9 +331,15 @@ export function loadBundle(
   const firestoreImpl = cast(firestore, Firestore);
   const resultTask = new LoadBundleTask();
   // eslint-disable-next-line @typescript-eslint/no-floating-promises
-  getSyncEngine(firestoreImpl).then(syncEngine =>
-    enqueueLoadBundle(firestoreImpl._queue, syncEngine, bundleData, resultTask)
-  );
+  getSyncEngine(firestoreImpl).then(async syncEngine => {
+    enqueueLoadBundle(
+      (await firestoreImpl._getConfiguration()).databaseInfo.databaseId,
```

Comment:
This style of code is pretty hard to debug (a lot of the magic happens inline). Do you mind extracting a local variable for this?

Reply:
Done.